indexbuilder.go

```diff
@@ -425,6 +429,16 @@ func (b *IndexBuilder) Add(doc Document) error {
 		}
 	}
 
+	// only compute language metadata for ambiguous extensions
+	if doc.Language == "" && len(enry_data.LanguagesByExtension[path.Ext(doc.Name)]) >= 2 {
+		c := doc.Content
+		// classifier is faster on small files without losing much accuracy
```

Comment:
how much of an impact did this have? did enry get stuck on larger files?

Reply:
It makes indexing noticeably slower if it fingerprints the entire file